frontend/packages/ceph-storage-plugin/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@console/ceph-storage-plugin",
+  "version": "0.0.0-fixed",
+  "description": "Ceph Storage - Persistent storage for kubernetes ",
+  "private": true,
+  "dependencies": {
+    "@console/plugin-sdk": "0.0.0-fixed",
+    "@console/shared": "0.0.0-fixed"
+  },
+  "consolePlugin": {
+    "entry": "src/plugin.tsx"
+  }
+}
```

Comment:
Nit-pick: every source should have one trailing newline.

Reply:
@vojtechszocs done